src/main/java/seedu/bigpp/ui/UI.java

```diff
@@ -113,18 +112,22 @@ public static void setMainMenuMode() {
     }
 
     public static void setViewerMode() {
+        visitedMenusStack.push(UIState.VIEWER);
```

Comment:
pushed menus into stack on setCommands, such that wrong commands will not create duplicates